docs/source/getstarted.rst

```diff
@@ -59,101 +38,55 @@ Install LeApp-To using these steps: ::
     sudo dnf copr enable evilissimo/leapp
     sudo dnf install leapp-cockpit
 
-Optional (Demo setup): ::
-        
-    sudo dnf install vagrant qemu libguestfs-tools-c libvirt libvirt-devel ruby-devel gcc qemu-kvm nmap libffi-devel
-
-
-Preparation & Running the demo
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-If the integration tests haven't been run, first install the testing
-instance of the CLI: ::
-
-    pipsi --bin-dir $PWD/bin install --python `which python2.7` $PWD/src
-
-The integration tests do this automatically, so this step can be skipped if
-those have already been run.
-
-Make sure the libvirt daemon is running and enabled: ::
-
-    for i in start enable; do sudo systemctl $i libvirtd; done
-
-The prototype requires Vagrant with some relevant plugins: ::
+    
+Enable and Start  Docker&Cockpit
+---------------------------
 
-    sudo vagrant plugin install ansible hitimes nio4r vagrant-libvirt
+    systemctl enable docker
+    systemctl enable cockpit
+    systemctl start docker
+    systemctl start cockpit
 
-And passwordless access to VM management operations: ::
-
-    sudo usermod -aG vagrant,libvirt $USER
-
-You will need to log out and back in again, or start a new user
-session with `su $USER`, to get the new group memberships to take
-effect.
-
-Finally, start all the demonstration VMs by running: ::
-
-    ln -s ../available/centos6-guest demo/vmdefs/enabled/
-    ln -s ../available/centos7-target demo/vmdefs/enabled/
-    sudo demo/vmctl.sh provision
-
-This script iterates over all the subdirectories of `demo/vmdefs/enabled` and runs
-`vagrant up --provision`.
-
-Note that the VMs shall be started by the same user who will run the
-demo, i.e. when using sudo to run the demo, one shall use sudo to
-start the VMs as well.
 
 Running the demonstration via the CLI
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-First, check that the source VM is showing the
-PHP admin login page at port 9000, while the target VM isn't
-running a HTTP server.
-
-The demo admin login credentials are:
-
 * Username: `root`
```

Comment:
These don't have an intro paragraph any more. Did you mean to delete them as well?